queries.sql

```diff
@@ -1,3 +1,56 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE (escape_attempts < 3) AND neutered;
+SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth < '2022-01-01';
```

Comment:
- [ ] It would be better if we delete all animals born after Jan 1st, 2022. In order to achieve that let us kindly change `<` to `>` because of the word `after`